<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="65247" started-at="2021-05-19T07:48:24Z" finished-at="2021-05-19T07:49:29Z">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunners.LoginRunnerTestNG@72b6cbcc]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Test" duration-ms="65247" started-at="2021-05-19T07:48:24Z" finished-at="2021-05-19T07:49:29Z">
      <class name="TestRunners.LoginRunnerTestNG">
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:TestRunners.LoginRunnerTestNG@72b6cbcc]" name="setUpClass" is-config="true" duration-ms="1581" started-at="2021-05-19T07:48:24Z" finished-at="2021-05-19T07:48:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunners.LoginRunnerTestNG@72b6cbcc]" name="runScenario" duration-ms="30237" started-at="2021-05-19T07:48:25Z" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2021-05-19T07:48:55Z">
          <params>
            <param index="0">
              <value>
                <![CDATA["Check login is succesful with valid credential"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Testing of the Logging Application]"]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.UnhandledAlertException">
            <message>
              <![CDATA[unexpected alert open: {Alert text : User or Password is not valid}
  (Session info: chrome=89.0.4389.114): User or Password is not valid
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LIE-CZ9NTC2', ip: '10.37.243.219', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_251'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 89.0.4389.114, chrome: {chromedriverVersion: 89.0.4389.23 (61b08ee2c5002..., userDataDir: C:\Users\kgajdosz\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:59464}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 69a0c48c9278c4b66074d3cc48cd6e03]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : User or Password is not valid}
  (Session info: chrome=89.0.4389.114): User or Password is not valid
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LIE-CZ9NTC2', ip: '10.37.243.219', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_251'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 89.0.4389.114, chrome: {chromedriverVersion: 89.0.4389.23 (61b08ee2c5002..., userDataDir: C:\Users\kgajdosz\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:59464}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 69a0c48c9278c4b66074d3cc48cd6e03
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:120)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:609)
at org.openqa.selenium.remote.RemoteWebDriver.close(RemoteWebDriver.java:442)
at stepDefininitions.LoginSteps.TernDown(LoginSteps.java:70)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.UnhandledAlertException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method status="PASS" signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunners.LoginRunnerTestNG@72b6cbcc]" name="runScenario" duration-ms="51099" started-at="2021-05-19T07:48:25Z" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2021-05-19T07:49:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA["Check login is succesful with valid credential"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Testing of the Logging Application]"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method status="PASS" signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunners.LoginRunnerTestNG@72b6cbcc]" name="runScenario" duration-ms="50877" started-at="2021-05-19T07:48:25Z" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2021-05-19T07:49:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA["Check login is succesful with valid credential"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Testing of the Logging Application]"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method status="PASS" signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunners.LoginRunnerTestNG@72b6cbcc]" name="runScenario" duration-ms="32866" started-at="2021-05-19T07:48:55Z" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2021-05-19T07:49:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA["Check login is succesful with valid credential"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Testing of the Logging Application]"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method status="PASS" signature="tearDownClass()[pri:0, instance:TestRunners.LoginRunnerTestNG@72b6cbcc]" name="tearDownClass" is-config="true" duration-ms="472" started-at="2021-05-19T07:49:28Z" finished-at="2021-05-19T07:49:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- TestRunners.LoginRunnerTestNG -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
